setup.py

```diff
@@ -32,7 +32,7 @@ def run_tests(self):
     packages=['language_detector'],
     maintainer='rmotr.com',
     tests_require=[
-        'pytest==2.9.1',
```

Comment:
I believe Santiago edited that as it was throwing an error because the Pytest came out with a later version which they didn't expect. 

Sent from my iPhone

> On Aug 24, 2016, at 12:55 PM, stevenpmartinez <notifications@github.com> wrote:
> 
> In setup.py:
> 
> > @@ -32,7 +32,7 @@ def run_tests(self):
> >      packages=['language_detector'],
> >      maintainer='rmotr.com',
> >      tests_require=[
> > -        'pytest==2.9.1',
> Why did you remove this line from the code?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>